### ISSUE TITLE
Having a `/` symbol breaks ns analysis

### ISSUE THREAD

**Issue by achesnais** [clojure-emacs/clj-refactor.el#342]
## Current behavior

When calling `cljr-clean-ns`, `/` by itself causes an exception. This only happens if it refers to the actual divide function (instead of when it's a namespace separator).

## Steps to reproduce the problem

Tested on a blank project. A simple expression like `(/ 10 5)` is enough to break `cljr-clean-ns`.

### clj-refactor.el version information

clj-refactor 2.3.0-SNAPSHOT (package: 20160621.902), refactor-nrepl 2.3.0-SNAPSHOT

### CIDER version information

CIDER 0.13.0 (package: 20160725.136) (California), nREPL 0.2.12
Clojure 1.8.0, Java 1.8.0_91

### Emacs version

24.5.1

### Operating system

OS X 10.11 "El Capitan"

**Comment by aiba** [comment]
This bug has been biting me too.

**Comment by aiba** [comment]
It looks like the bug was fixed in clojure-emacs/refactor-nrepl#166, but a new snapshot artifact has not been deployed yet.  Until they do, this has been my workaround:

```clj
(when-let [f (resolve 'refactor-nrepl.core/prefix)]
  (alter-var-root
   f
   (fn [_]
     (fn [fully-qualified-name]
       (if (re-find #"/" (str fully-qualified-name))
         (when-let [ns-name (-> fully-qualified-name str (.split "/") first)]
           (clojure.string/replace ns-name #"^:" ""))
         (let [parts (-> fully-qualified-name str (.split "\\.") butlast)]
           (when (seq parts)
             (clojure.string/join "." parts))))))))
```

**Comment by expez** [comment]
New snapshot is out.  Sorry about that.  I thought I'd already deployed the fix.
